backend/src/graphql/resolver/GdtResolver.ts

```diff
@@ -35,6 +33,25 @@ export class GdtResolver {
     }
   }
 
+  @Authorized([RIGHTS.GDT_SUM])
+  @Query(() => Number)
+  async gdtSum(@Ctx() context: any): Promise<number | null> {
```

Comment:
I vote for gdtBalance in this case - better name, more descriptive and aligned with `balance`

Reply:
Also: Why expose this to the GraphQL interface? Do we need that? Can't it just be part of `balance` without external exposure?

Reply:
We expose the gst transaction list but not the gdt balance? Does not make sense for me. May be we have an use case in the future where we want only query the gdt balance. Here it is